OpenSim/Common/AbstractDataTable.h

```diff
@@ -56,6 +61,21 @@ class IncorrectNumColumns : public InvalidRow {
     }
 };
 
+class IncorrectNumRows : public InvalidColumn {
+public:
+    IncorrectNumRows(const std::string& file,
+                     size_t line,
+                     const std::string& func,
+                     size_t expected,
+                     size_t received) :
+        InvalidColumn(file, line, func) {
+        std::string msg = "expected = " + std::to_string(expected);
+        msg += " received = " + std::to_string(received);
```

Comment:
The message would also be easier to interpret with a comma (or other non-space delimiter) before "received".